go.mod

```diff
@@ -18,3 +18,14 @@ require (
 	google.golang.org/protobuf v1.26.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 )
+
+require (
+	github.com/golang/protobuf v1.5.2 // indirect
```

Comment:
why need this? 
specify indirect lib version may lead build failed.

Reply:
This is to enable [module graph pruning](https://golang.org/ref/mod#graph-pruning) and [lazy module loading](https://golang.org/ref/mod#lazy-loading) supported by Go 1.17 and higher. 

It was automatically added when I run `go mod tidy -go=1.17`.

Reply:
Sorry, it will affect the lower go versions, we cannot accept this change, including ioutil->io.